### ISSUE TITLE
Add support for map.fitbounds

### ISSUE THREAD

**Issue by enriqueav** [gmplot/gmplot#77]
When it is difficult to calculate the correct zoom level to show an specific area of the map, google maps allow the use of Map.fitBounds. It basically needs only the NE and SW coordinates:
https://developers.google.com/maps/documentation/javascript/reference/3/map#Map.fitBounds

I have added the support for it in my fork, and I would like to do it in the main branch if approved. My only questions is. Should this be added as a separate function? something like `def fit_bounds(self, nelat, nelng, swlat, swlng):` or as a different way to initialize the map. That means providing NE and SW instead of `center_lat, center_lng, zoom`.

**Comment by adeharo9** [comment]
Agreed, this feature would make fitting the map in the desired bounds way easier than it is right now.

**Comment by keithstellyes** [comment]
I would also find this highly useful

**Comment by enriqueav** [comment]
@keithstellyes 
You may want to take a look at this:
https://github.com/enriqueav/gmplot/blob/master/gmplot/gmplot.py#L173

```python
# Call to fit_bounds
# https://developers.google.com/maps/documentation/javascript/reference/map#Map.fitBounds
gmap.fit_bounds(north, east, south, west)
```

**Comment by keithstellyes** [comment]
Right as you said that I found this commit and got it working:

https://github.com/vgm64/gmplot/pull/12/commits/2b5b2d634534dd2b31fc8eae566bffdc22a579d5

However that code in that PR is slightly bugged in that SW is supposed to go first, NE. 

Thanks either way though! Appreciate it :)

**Comment by frslm** [comment]
I've just merged this into master in #153, you can set it by passing in a bounds literal when creating a `GoogleMapPlotter` object:
```python
import gmplot

fit_bounds = {'north':37.832285, 'south': 37.637336, 'west': -122.520364, 'east': -122.346922}
gmap = gmplot.GoogleMapPlotter(37.766956, -122.438481, 13, fit_bounds=fit_bounds)
```
